Dynatrace/dynatrace-operator
# Set OneAgent readiness probe value based on image Healthcheck

## Description

Replace the old hardcoded OneAgent readiness probe. Now we get the `Healthcheck`  field from OneAgents image
and set it as readiness probe.

To do so, `Healthcheck` is first set in dynakube status and then reconciled to the OneAgent pod as readiness probe.

In case there is not `Healthcheck` field in the image, the readiness probe is not set.

## How can this be tested?

- `make deploy` this version of the operator
- Install a dynakube e.g CloudNativeFullStack
- Check that `Healthcheck` field is set on Dynakube status
- Check that this value is then set on each OneAgent pods as readiness probe

## Checklist
